src/mbgl/map/map.cpp

```diff
@@ -113,7 +127,6 @@ void Map::start(bool startPaused) {
 
         // Remove all of these to make sure they are destructed in the correct thread.
         style.reset();
-        workers.reset();
```

Comment:
This crashes in Debug mode since the Worker threads are now joined from the wrong thread.